Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Cloud_IAM_User_Access_Investigation.yml

```diff
@@ -0,0 +1,1133 @@
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1350
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "10":
+    id: "10"
+    taskid: 46d8a585-dbc1-44b6-8653-6ad06f7f1976
+    type: title
+    task:
+      id: 46d8a585-dbc1-44b6-8653-6ad06f7f1976
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1660
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "11":
+    id: "11"
+    taskid: 4de4fb9a-ee3d-49dd-86f4-62f4d42061ec
+    type: playbook
+    task:
+      id: 4de4fb9a-ee3d-49dd-86f4-62f4d42061ec
+      version: -1
+      name: Cloud IAM Enrichment - Generic
+      description: This playbook is responsible for collecting and enriching data on Identity Access Management (IAM) in cloud environments (AWS, Azure, and GCP).
+      playbookName: Cloud IAM Enrichment - Generic
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "5"
+    scriptarguments:
+      GCPProjectName:
+        complex:
+          root: inputs.GCPProjectName
+      cloudIdentityType:
+        complex:
+          root: inputs.cloudIdentityType
+      cloudProvider:
+        complex:
+          root: inputs.cloudProvider
+      username:
+        complex:
+          root: inputs.username
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 0
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 2
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "12":
+    id: "12"
+    taskid: d121fca8-3d36-47c8-8fec-4767e752b0cb
+    type: regular
+    task:
+      id: d121fca8-3d36-47c8-8fec-4767e752b0cb
+      version: -1
+      name: Investigate collected data
+      description: |2-
+         To determine if this is a true positive event,  review the operations performed by the access key and the user in the recent time frame.
+        Search for the following items:
+        Investigate the operation performed by the access key and examine the executed operations, by who it was executed, on which resource, and the operation status.
+        Investigate operations performed by the user and examine the executed operations, by who it was executed, on which resource, and the operation status.
```

Comment:
If these are operations performed by the user, then "by who it was executed"  - wasn't it the user?